README.md

```diff
@@ -18,25 +18,28 @@
 
 Plone backend [Docker](https://docker.com) images using Python 3 and [pip](https://pip.pypa.io/en/stable/).
 
-> **NOTE**:
-> These images are not officially supported by the Plone Community for Plone 5.
-> They will become the official ones for the upcoming Plone 6 release (together with [plone.frontend](https://github.com/plone/plone-frontend)).
-> These image are **not** buildout based!
+**Note:**
+These are the official images for the Plone 6 release, together with [plone.frontend](https://github.com/plone/plone-frontend).
```

Comment:
```suggestion
These are the official images for the Plone 6 release, together with [plone-frontend](https://github.com/plone/plone-frontend).
```

Reply:
This change was not committed. Unresolving.